src/main/java/uk/gov/hmcts/sptribs/caseworker/util/DocumentListUtil.java

```diff
@@ -78,7 +78,7 @@ private static List<DynamicListElement> getDynamicListElements(List<CaseworkerCI
 
     public static DynamicMultiSelectList prepareDocumentList(final CaseData data, String baseUrl) {
         List<CaseworkerCICDocument> docList = prepareList(data);
-        String apiUrl = baseUrl + "/cases/documents/%s/binary";
+        String apiUrl = "https://manage-case.demo.platform.hmcts.net/documents/%s/binary";
         List<DynamicListElement> dynamicListElements = new ArrayList<>();
```

Comment:
Do we really want to hardcode a URL like this?

Reply:
for testing